### PR TITLE
Swedish translation update

### PR DESCRIPTION
**Pull request by Mr-Bajs**

### CODE REVIEW

**Comment by Mr-Bajs** [comment]
@zenseii I'll leave it at that for now. I'll wait for your review whenever you have the time.

Uncertain about Pathfinding. Stigletning isn't very good. Vägvisning is probably better,

File: files/lang/sv.po [hunk 1]
```diff
@@ -883,10 +875,10 @@ msgid "Right Turret"
 msgstr "Höger Torn"
 
 msgid "The %{name} fires with the strength of %{count} Archers"
-msgstr ""
+msgstr "%{name} avfyrar med styrkan av %{count} Bågskyttar"
 
 msgid "each with a +%{attack} bonus to their attack skill."
-msgstr ""
+msgstr "varje med en +%{attack}bonns till deras Anfallsförmåga."
```
[the remark this answers is not in the document]
Comment (Mr-Bajs) [reply files/lang/sv.po]: Should be one word

File: files/lang/sv.po [hunk 2]
```diff
@@ -389,9 +386,8 @@ msgstr ""
 msgid "Grid"
 msgstr "Rutnät"
 
-#, fuzzy
 msgid "Shadow Movement"
-msgstr "Fortsätt förflyttning"
+msgstr "Beskuggad Förflyttning"
```
[the remark this answers is not in the document]
Comment (Mr-Bajs) [reply files/lang/sv.po:390]: I know. It's a bad translation. And the original Shadow Movement is sort of weird in it self. Though the meny does have a image that shows and explains clearly. Changed into Skuggförflyttning non the less.

File: files/lang/sv.po [hunk 3]
```diff
@@ -9339,7 +9338,7 @@ msgid ""
 "Increases the defense skill of the targeted creatures.  This is an improved "
 "variant of Stoneskin."
```
[the remark this answers is not in the document]
Comment (Mr-Bajs) [reply files/lang/sv.po]: @zenseii
Been using Poedit instead lately so i guess most of these minor errors won't occur.

File: files/lang/sv.po [hunk 4]
```diff
@@ -6614,29 +6604,25 @@ msgstr ""
...
 "procent."
 
-#, fuzzy
 msgid "%{skill} increases your hero's troops morale by %{count}."
-msgstr "Höjer din hjältes tur med %{count}."
+msgstr "%{skill} höjer din armés stridsmoral med %{count}."
 
 msgid "%{skill} allows your hero to learn third level spells."
-msgstr "%{skill} låter din hjälte lära sig tredje nivåns besvärjelser."
+msgstr "%{skill} låter din hjälte att lära sig %{count}:e nivåns besvärjelser."
```
[the remark this answers is not in the document]
Comment (Mr-Bajs) [reply files/lang/sv.po:6611]: True but i checked and int translates well.
vidom
Wisdom has a count functinality so it works.

File: files/lang/sv.po [hunk 5]
```diff
@@ -6614,29 +6604,25 @@ msgstr ""
...
 
-#, fuzzy
 msgid "%{skill} regenerates one additional spell point per day to your hero."
 msgid_plural ""
 "%{skill} regenerates %{count} additional spell points per day to your hero."
 msgstr[0] ""
-"Du får kontroll över en kristallgruva. Den kommer att förse dig med %{count} "
-"enheter av kristaller per dag."
+"%{skill} återhämtar ytterligare %{count} av hjältens besvärjelsepoäng per dag."
```
[the remark this answers is not in the document]
Comment (Mr-Bajs) [reply files/lang/sv.po:6623]: I works. Also there is no need to change this again manually due to the coming fix of https://github.com/ihhub/fheroes2/pull/6905 The english text should be changed to include %{count} so no wrongly translations of Mysticism lingers on.

myst

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Mr-Bajs , great thanks for this update!

**Pull request merged**